src/test/java/uk/gov/digital/ho/hocs/entrypoint/integration/MigrationQueueListenerIntegrationTest.java

```diff
@@ -73,4 +73,10 @@ public void consumeMessageFromMigrationQueueWithPOGRCaseType() {
         String validMessage = TestFileReader.getResourceFileAsString("migration/validMigrationPOGR.json");
         amazonSQSAsync.sendMessage(queueUrl, validMessage);
     }
+
+    @Test
+    public void consumeMessageFromMigrationQueueWithTOCaseType() {
```

Comment:
What is this testing/how does it work. It seems to just put the message on the queue and not verify anything?

Reply:
Good point, there's a comment at the top of this file saying that these tests aren't currently used so I've removed this